bin/smart-proxy-win-service

```diff
@@ -1,40 +1,38 @@
-#!/usr/bin/env ruby
-
 $LOAD_PATH.unshift(*Dir[File.expand_path("../../lib", __FILE__), File.expand_path("../../modules", __FILE__)])
 
-raise "win-service executable can only be used as a service in windows environment" unless RUBY_PLATFORM =~ /mingw/
-
-require 'smart_proxy'
+require 'rubygems'
 require 'win32/daemon'
+require 'smart_proxy'
+
 include Win32
 
-begin
-  logfile = "../../tmp/proxy.log"
-  $stdout.reopen(logfile, "a")
-  $stdout.sync = true
-  $stderr.reopen($stdout)
-  puts "#{Time.now}: Service is starting"
+def logger
+  @logger ||= Logger.new(File.expand_path("win-service.log", File.expand_path("../../log", __FILE__)))
```

Comment:
I think the reason for a fail safe is that it is fail save :). IMHO it shout be changed in way to go directly to the smart-proxy root folder.

Also, can explain think why `@logger` is not there? Is this not read from settings.yml?